cdap-app-fabric/src/test/java/co/cask/cdap/ConfigTestApp.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap;
+
+import co.cask.cdap.api.Config;
+import co.cask.cdap.api.app.AbstractApplication;
+import co.cask.cdap.api.data.stream.Stream;
+import co.cask.cdap.api.dataset.table.Table;
+import co.cask.cdap.api.worker.AbstractWorker;
+import com.clearspring.analytics.util.Preconditions;
```

Comment:
while you're at it, perhaps change the import in following two files as well.
`AdapterDefinition.java`
`AdapterService.java`